Sources/PackageDescription/Version.swift

```diff
@@ -56,7 +56,11 @@ extension Version: Hashable {
         if let build = buildMetadataIdentifier {
             result = (result &* mul) ^ UInt64(build.hashValue)
         }
-        return Int(bitPattern: UInt(result))
+        #if arch(i386) || arch(arm)
```

Comment:
A more general fix might be to query `MemoryLayout<Int>.size` to see if truncation of 64-bit values is required.